components/inputs/docs/input-date-time.md

```diff
@@ -1,24 +1,46 @@
 # Date & Time Inputs
 
-## Date Inputs
+<!-- docs: demo
+align:flex-start
+autoSize:false
+size:xlarge
+-->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-date.js';
+  import '@brightspace-ui/core/components/inputs/input-time.js';
+</script>
+<d2l-input-date label="Date Input"></d2l-input-date>
+<d2l-input-time label="Time Input"></d2l-input-time>
+```
+
+## Date Input [d2l-input-date]
 
 The `<d2l-input-date>` component consists of a text input field for typing a date and an attached calendar (`<d2l-calendar>`) dropdown. The dropdown opens on click of the text input, or on enter or down arrow press if the text input is focused. It displays the `value` if one is specified, or a placeholder if not, and reflects the selected value when one is selected in the `d2l-calendar` or entered in the text input.
 
-![example screenshot of date input](../screenshots/date.gif?raw=true)
+Note: All `*value` properties should be in ISO 8601 calendar date format (`YYYY-MM-DD`) and should be [localized to the user's timezone](#timezone) (if applicable).
 
+<!-- docs: start hidden content -->
+![example screenshot of date input](../screenshots/date.gif?raw=true)
+<!-- docs: end hidden content -->
+
+<!-- docs: demo live
+name:d2l-input-date
+align:flex-start
+autoSize:false
+defaults:{ "label": "Birthdate" }
+size:xlarge
+-->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/inputs/input-date.js';
 </script>
-<d2l-input-date
-  label="Start Date"
-  value="2020-11-20">
+<d2l-input-date>
```

Comment:
It's unfortunate that when viewing on GitHub the inline code example isn't going to contain any of the attributes you need to get it to work.

Would it be possible for the docs site to just ignore any attributes that are set here? Probably not easy eh.

Reply:
Or I guess even better: use the attributes provided here as the defaults so that `defaults: { "label": "Birthdate" }` didn't need to be passed in as part of the metadata.

Reply:
I'll take a look at that. It's a bit more difficult since they're initially just strings and not code we can pull the attributes out of, but there's likely some regex we could use.